src/sentry/eventtypes/error.py

```diff
@@ -52,7 +80,10 @@ def compute_title(self, metadata):
             if value:
                 title += f": {truncatechars(value.splitlines()[0], 100)}"
 
-        return compute_title_with_tree_label(title, metadata)
+        if not metadata.get("hide_tree_labels"):
```

Comment:
nit: My brain has trouble with double negatives. Would it be more straight forward to have a field `metadata["show_tree_labels"]`?

Reply:
in the frontend is `displayTitleWithTreeLabel` positive :)